2] loto/main.py

```diff
@@ -0,0 +1,59 @@
+import random
+
+
+def get_computer_numbers():
+    """Get 6 random numbers from the range of 1 – 49
+
+    Returns:
+        list: 6 random numbers from the range of 1 – 49
+    """
+    numbers = []
+    while len(numbers) != 6:
+        n = random.randint(40, 49)
+        if n not in numbers:
```

Comment:
tady taky jeste poznaka = kdyz pouzivas in operator tak je lepsi mit set a ne list. Je tam jina casova komplexita na ziskni toho prvnku - list je linearni, set je konstantni.  

Tady je ten list malej takze to tolik nevadi